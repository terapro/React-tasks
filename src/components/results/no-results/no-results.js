import React from 'react';
import PropTypes from "prop-types";

export const NoResults = ({phrase, type}) => (
  <div className={'no-results'}>
    By request &quot;<span>{phrase}</span>&quot; in <span>{type}</span> nothing was found. <br/>
    Don`t worry, try to make another request!
  </div>
);

NoResults.propTypes = {
  phrase: PropTypes.string.isRequired,
  type: PropTypes.string.isRequired
};
