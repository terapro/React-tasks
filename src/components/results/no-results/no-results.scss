.no-results {
  min-height: 110px;
  width: inherit;
  text-align: center;
  font-size: 40px;
  padding-top: 200px;
}

.no-results span {
  color: #0056b3;
}