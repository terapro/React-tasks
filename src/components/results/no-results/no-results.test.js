import React from 'react';
import {NoResults} from 'src/components/results/no-results/no-results.js';
import {shallow} from 'enzyme';

describe('<NoResults />', () => {
  describe('Rendering', () => {
    it('should render the component', () => {
      const wrapper = shallow(
        <NoResults/>
      );
      expect(wrapper).toMatchSnapshot();
    });
  });
});
