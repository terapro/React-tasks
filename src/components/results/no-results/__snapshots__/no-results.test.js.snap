// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResults /> Rendering should render the component 1`] = `
<div
  className="no-results"
>
  No films found
</div>
`;
